src/main.rs

```diff
@@ -53,12 +51,13 @@ impl Stack {
     }
 }
 
+const DIMENSION: usize = 32;
```

Comment:
Allow this to be specified through the commandline. A future state may be to allow it to be specified for each gif.

Reply:
Can this be a `u16`, that seems to be the only way it's used